log-elasticsearch/log-elasticsearch.cabal

```diff
@@ -24,16 +24,21 @@ Source-repository head
   Location: https://github.com/scrive/log.git
 
 library
-  exposed-modules:     Log.Backend.ElasticSearch
-                       Log.Backend.ElasticSearch.Internal
+  exposed-modules:     Log.Backend.ElasticSearch.V1
+                       Log.Backend.ElasticSearch.V1.Internal
+                       Log.Backend.ElasticSearch.V1.Lens
+                       Log.Backend.ElasticSearch.V5
+                       Log.Backend.ElasticSearch.V5.Internal
+                       Log.Backend.ElasticSearch.V5.Lens
+                       Log.Backend.ElasticSearch
```

Comment:
I'm not sure, adding one would be safer indeed. Also I'll add deprecated shim modules to `log` too.